src/python/pants/backend/python/dependency_inference/import_parser.py

```diff
@@ -59,6 +59,18 @@ def visit_ImportFrom(self, node):
         for alias in node.names:
             self.explicit_imports.add("{}.{}".format(abs_module, alias.name))
 
+    def visit_Call(self, node):
+      # Handle __import__("string_literal").  This is commonly used in __init__.py files,
+      # to explicitly mark namespace packages.
+      if (
+          isinstance(node.func, ast.Name)
+          and node.func.id == "__import__"
+          and len(node.args) == 1
+      ):
+          if sys.version_info[0:2] < (3, 8) and isinstance(node.args[0], ast.Str):
```

Comment:
Note than in Py2, this can either by bytes or Unicode. A couple lines lower, we check for this and decode to emulate ensure_text(). I'm not certain that's necessary here, but would be worth including a byte string in the unit test for Py2.

Reply:
> Great idea. (I'm on my phone) could you please double check if the help message for string imports is still accurate? I think we may have used this statement as an example.

I don't see that anywhere.

Reply:
Added new tests (which also exposed a bug).